builtin/tag.c

```diff
@@ -425,9 +425,13 @@ int cmd_tag(int argc, const char **argv, const char *prefix)
 		die(_("--merged and --no-merged option are only allowed with -l"));
 	if (cmdmode == 'd')
 		return for_each_tag_name(argv, delete_tag);
-	if (cmdmode == 'v')
+	if (cmdmode == 'v') {
+		if (fmt_pretty)
+			verify_ref_format(fmt_pretty);
```

Comment:
I think we want to move these lines to the commit where we actually start passing `fmt_pretty` to `verify_and_format_tag`